queries/odin/rainbow-delimiters.scm

```diff
@@ -0,0 +1,83 @@
+(parameters
+  "(" @delimiter
+  ")" @delimiter @sentinel) @container
+
+(tuple_type
+  "(" @delimiter
+  ")" @delimiter @sentinel) @container
+
+(parenthesized_expression
+  "(" @delimiter
+  ")" @delimiter @sentinel) @container
+
+(polymorphic_parameters
+  "(" @delimiter
+  ")" @delimiter @sentinel) @container
+
+(polymorphic_type
+  "(" @delimiter
+  ")" @delimiter @sentinel) @container
+
+(attribute
+  "(" @delimiter
+  ")" @delimiter @sentinel) @container
+
+(call_expression
+  "(" @delimiter
+  ")" @delimiter @sentinel) @container
+
+(index_expression
+  "[" @delimiter
+  "]" @delimiter @sentinel) @container
+
+(slice_expression
+  "[" @delimiter
+  "]" @delimiter @sentinel) @container
+
+(block
+  "{" @delimiter
+  "}" @delimiter @sentinel) @container
+
+(switch_statement
+  "{" @delimiter
+  "}" @delimiter @sentinel) @container
+
+(array_type
+  "[" @delimiter
+  "]" @delimiter @sentinel) @container
+
+(struct
+  ["{" "(" "["] @delimiter
+  ["}" ")" "]"] @delimiter @sentinel) @container
+
+(map_type
+  "[" @delimiter
+  "]" @delimiter @sentinel) @container
+
+(map
+  ["{" "["] @delimiter
+  ["}" "]"] @delimiter @sentinel) @container
```

Comment:
Same problem as with `struct`. How about this?

```suggestion(map
  "[" @delimiter
  "]" @delimiter
  "{" @delimiter
  "}" @delimiter @sentinel) @container
```

The only `map` example in your code has all delimiters. Is it possible in Odin to have only of of the two sets?